framework/src/media/audio/audio_manager.c

```diff
@@ -212,6 +208,8 @@ static audio_manager_result_t find_audio_card(audio_card_type_t card_type)
 		g_actual_audio_out_card_id = get_actual_audio_out_card_id();
 	}
 
+	get_hardware_params(&card[card_id], card_type);
```

Comment:
Also I put this inside of loop to get all h/w params.
In this case we can get only one h/w params.

Reply:
only one h/w params means only one card's param,
find -> get params -> find next card -> get params ....

Reply:
Okay, I'll update.